tech-far-hub/src/pages/template-lifecycle.tsx

```diff
@@ -1,11 +1,9 @@
-import type { HeadFC, PageProps } from "gatsby";
```

Comment:
Realized I'd missed these in a previous PR --- I should probably add a linter at this point.